tcms/testruns/tests/test_permissions.py

```diff
@@ -56,3 +58,39 @@ def verify_post_with_permission(self):
         self.assertRedirects(
             response,
             reverse('testruns-get', args=[self.test_run.pk]))
+
+
+class TestRunTestCase(tests.PermissionsTestCase):
+    permission_label = 'testruns.add_testrun'
+    url = reverse('testruns-new')
+    http_method_names = ['post']
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.test_run = factories.TestRunFactory(summary='Old summary')
+        cls.url = reverse('testruns-edit', args=[cls.test_run.pk])
+        cls.new_build = factories.BuildFactory(name='FastTest', product=cls.test_run.plan.product)
+
+        intern = factories.UserFactory(username='intern',
+                                       email='intern@example.com')
+        cls.post_data = {
+            'summary': 'New run summary',
+            'build': cls.new_build.pk,
+            'manager': cls.test_run.manager.email,
+            'default_tester': intern.email,
+            'notes': 'New run notes',
+        }
```

Comment:
This `post_data` doesn't seem correct. Why did you delete the previous code and substitute with something completely unrelated ?  See previous comments in this section.